R/fill.R

```diff
@@ -0,0 +1,22 @@
+#' Fill in missing values with the previous or following value
+#'
+#' `vec_fill()` fills gaps of missing values with the previous or following
+#' non-missing value.
+#'
+#' @param x A vector
+#' @param direction Direction in which to fill missing values. Must be either
+#'   `"down"` or `"up"`.
+#' @param leading A logical specifying if leading missing values should be
```

Comment:
I find `downup`/`updown` easier to understand, and it has the advantage of matching the existing `tidyr::fill()` syntax which some people will be familiar with.